foxpuppet/windows/browser/notifications/addons.py

```diff
@@ -22,6 +22,8 @@ class AddOnInstallConfirmation(BaseNotification):
 
     _addon_name_locator = (
         By.CSS_SELECTOR, '#addon-install-confirmation-content label')
+    _web_ext_header_locator = (By.ID, 'addon-webext-perm-header')
+    _web_ext_name_locator = (By.CSS_SELECTOR, '.addon-webext-name')
```

Comment:
I think we should just reassign the value of `_addon_name_locator` within the conditional logic. Once all supported Firefox versions use the same locator we just remove the conditional block and revert to a single value for the locator.